plugins/catalog-backend/fixtures/one_component.yaml

```diff
@@ -4,3 +4,5 @@ metadata:
   name: component3
```

Comment:
Can this be updated to something humanly understandable as well?

Reply:
Also, why not add `description`?

Reply:
Well I mainly wanted to unbreak them. These will all soon be thrown away entirely, and be replaced with an `example-entities.yaml` file instead.